recipes/conda-forge-tick/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set name = "conda-forge-tick" %}
+{% set version = "2024.8.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/regro/cf-scripts/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 1c85476b0b0856c621cc1fb8f8ce5f9053a683ca5007c4159a82271e417e0f8d
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.11
+    - pip
+    - setuptools
+    - setuptools_scm >=8
+  run:
+    - python >=3.11
+    - attrs
+    - backoff
+    - cachecontrol
+    - cachetools
+    - click
+    - conda
+    - conda-lock
+    - conda-forge-feedstock-check-solvable >=0.4.0
+    - conda-forge-pinning
+    - conda-libmamba-solver
+    - conda-forge-metadata >=0.3.0
+    - conda-package-handling
+    - conda-smithy >=3.34.1
+    - conda-build >=3.24.0
+    - curl
+    - depfinder
+    - distributed
+    - feedparser
+    - frozendict
+    - git
+    - github3.py
+    - grayskull >=2.5
+    - jinja2
+    - lockfile
+    - mamba >=0.23
+    - msgpack-python
+    - networkx !=2.8.1
+    - numpy
+    - packaging
+    - psutil
+    - pydantic
+    - pydantic-extra-types
+    - pygithub
+    - pymongo
+    - pynamodb
+    - python-dateutil
+    - python-graphviz
+    - python-rapidjson
+    - rattler-build
+    - requests
+    - ruamel.yaml
+    - ruamel.yaml.jinja2
+    - scipy
+    - stopit
+    - streamz
+    - tar
+    - tqdm
+    - wget
+    - wurlitzer
+    - yaml
+
+test:
+  imports:
+    - conda_forge_tick
+  commands:
+    - python -c "import conda_forge_tick; assert conda_forge_tick.__version__ == '{{ version }}'"
+
+about:
+  home: https://github.com/regro/cf-scripts
+  summary: 'Simple, fast, extensible JSON encoder/decoder for Python'
```

Comment:
Summary looks to be left over from the simplejson example recipe

Reply:
arg every time!

Reply:
fixed!